models/unet-2d-nuclei-broad/UNet2DNucleiBroad.model.yaml

```diff
@@ -41,37 +41,37 @@ outputs:
 
 prediction:
     preprocess:
-        - spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
+        - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/master/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
```

Comment:
I am not really convinced by this yet; now we are just specifying `master` and don't pin to a commit or version.
`master` could be replaced by the commit hash or version (I think both would still be valid urls?!), but this would be 
a bit cumbersome. One more minor point: we rely on the github url structure.
If we just have `repo_url:path_in_repo:commit_or_version` then it would work for any git hosted project.

Reply:
> master could be replaced by the commit hash or version (I think both would still be valid urls?!), but this would be

yes, indeed, we can easily check that it's not master, checking if it is a version string is a good idea as well, the full hash should be allowed in any case.

> If we just have repo_url:path_in_repo:commit_or_version then it would work for any git hosted project.

I like that it is a valid url, that you can just copy into your browser, we can easily add other git hosts in the future and adapt this scheme... there are only so many and only so many different ones + then no one needs to learn this new schema of ours